test/bdd/pkg/waci/waci_issuance_didcomm_v1_sdk_steps.go

```diff
@@ -0,0 +1,772 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package waci
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/cenkalti/backoff/v4"
+	"github.com/cucumber/godog"
+	"github.com/google/uuid"
+
+	didexClient "github.com/hyperledger/aries-framework-go/pkg/client/didexchange"
+	issuecredentialclient "github.com/hyperledger/aries-framework-go/pkg/client/issuecredential"
+	"github.com/hyperledger/aries-framework-go/pkg/client/outofband"
+	"github.com/hyperledger/aries-framework-go/pkg/common/log"
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/common/service"
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/protocol/decorator"
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/protocol/issuecredential"
+	"github.com/hyperledger/aries-framework-go/pkg/doc/cm"
+	"github.com/hyperledger/aries-framework-go/pkg/doc/verifiable"
+	"github.com/hyperledger/aries-framework-go/test/bdd/pkg/context"
+	bddDIDExchange "github.com/hyperledger/aries-framework-go/test/bdd/pkg/didexchange"
+)
+
+// TODO (#3143) Refactor these tests to reduce code duplication between the V1 and V2 tests.
```

Comment:
I will address this TODO in my next PR. I wanted to do it in a separate PR since it'll involve modifying the existing WACI Issuance DIDComm V2 tests a lot and this PR is already pretty big.